src/App.js

```diff
@@ -1,23 +1,29 @@
-import logo from './logo.svg';
-import './App.css';
-
+import React, { useContext } from "react";
+import "./App.css";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Login from "./pages/Login";
+import Signup from "./pages/SignUp";
+import Header from "./components/Common/Header/Header";
+import ThemeContext from "./context/ThemeContext";
+import Landing from "./pages/Landing";
+import PrivateRoute from "./components/Private/PrivateRoute";
+import Dash from "./pages/Dash";
```

Comment:
if it's for dashboard let's rename it to dashboard so it makes sense

Reply:
Okay